solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -775,10 +777,12 @@ contract RandomBeacon is Ownable {
     /// @param ids IDs of banned operators.
     /// @param banDuration Duration of the ban period in seconds.
     function banFromRewards(uint32[] memory ids, uint256 banDuration) internal {
```

Comment:
`approveDkgResult` is the only place where this function is called. Can we get rid of `banFromRewards` and just call `sortitionPool.banRewards(ids, banDuration)`, without try-catch, from `submitDkgResult` directly?

Reply:
See https://github.com/keep-network/keep-core/pull/2729/commits/ef1075cc3260578686539f43fba5bd11fc6b5a0a